src/app/mail/mail-sidebar/compose-mail/compose-mail.component.ts

```diff
@@ -895,4 +892,22 @@ export class ComposeMailComponent implements OnInit, AfterViewInit, OnDestroy {
       this.closeOSK();
     }
   }
+
+  onFilesDrop(event) {
+    if (!this.draftMail || !this.draftMail.id) {
+      this.updateEmail();
+    }
+    const files = event.dataTransfer.files;
+    for (let i = 0; i < files.length; i++) {
+      const file = files.item(i);
+      if (!/^image\//.test(file.type)) {
+        this.uploadAttachment(file, false);
+      }
+    }
+  }
+
+  onDragover(event) {
+    event.stopPropagation();
+    event.preventDefault();
```

Comment:
There is no need to prevent `dragover` event because we are not doing anything with `dragover` but instead prevent `drop` event.

Reply:
@Waseem826 its for preventing the chrome to open dragged and dropped files.

Reply:
This didn't fix the issue. You need to prevent `drop` event as I suggested in my comment above. It works fine.

Reply:
@Waseem826 would  you please check on both chrome and firefox?
without prevention of events in dragover event, chrome opens dragged & dropped files.

Reply:
@Waseem826 just a moment, double checking again.